src/main/java/lotto/entity/Rank.java

```diff
@@ -0,0 +1,63 @@
+package lotto.entity;
+
+import java.math.BigDecimal;
+
+
+public enum Rank {
+    FIRST(6, BigDecimal.valueOf(2_000_000_000)),
+    SECOND(5, BigDecimal.valueOf(30_000_000)),
+    THIRD(5, BigDecimal.valueOf(1_500_000)),
```

Comment:
보너스 번호 일치 여부도 추가하면 어떨까요?
```suggestion
    SECOND(5, true, BigDecimal.valueOf(30_000_000)),
    THIRD(5, false, BigDecimal.valueOf(1_500_000)),
```